eatin-client/src/generated/graphql.tsx

```diff
@@ -70,6 +155,32 @@ export type CreateUserMutationVariables = Exact<{
 
 export type CreateUserMutation = { __typename?: 'Mutation', createUser: { __typename?: 'Users', id: string, firstname: string, lastname: string, email: string, phone: string, gender: string, birthdate: any, country: string } };
 
+export type GetAllRecipesQueryVariables = Exact<{ [key: string]: never; }>;
+
+
+export type GetAllRecipesQuery = { __typename?: 'Query', recipes: Array<{ __typename?: 'Recipes', index: string, recipe_title: string, url: string, record_health: string, vote_count: string, rating: string, description: string, cuisine: string, course: string, diet: string, prep_time: string, cook_time: string, ingredients: string, instructions: string, author: string, tags: string, category: string, image: string, difficulty: string }> };
+
+export type GetRecipeByIdQueryVariables = Exact<{
+  index: Scalars['String'];
+}>;
+
+
+export type GetRecipeByIdQuery = { __typename?: 'Query', recipe: { __typename?: 'Recipes', index: string, recipe_title: string, url: string, record_health: string, vote_count: string, rating: string, description: string, cuisine: string, course: string, diet: string, prep_time: string, cook_time: string, ingredients: string, instructions: string, author: string, tags: string, category: string, image: string, difficulty: string } };
+
+export type GetTopRatedRecipesByCategoryQueryVariables = Exact<{
+  category: Scalars['String'];
+}>;
+
+
+export type GetTopRatedRecipesByCategoryQuery = { __typename?: 'Query', topRecipesByCategory: Array<{ __typename?: 'Recipes', index: string, recipe_title: string, url: string, record_health: string, vote_count: string, rating: string, description: string, cuisine: string, course: string, diet: string, prep_time: string, cook_time: string, ingredients: string, instructions: string, author: string, tags: string, category: string, image: string, difficulty: string }> };
+
+export type GetTopRatedRecipesByCuisineQueryVariables = Exact<{
+  cuisine: Scalars['String'];
+}>;
+
+
+export type GetTopRatedRecipesByCuisineQuery = { __typename?: 'Query', topRecipesByCuisine: Array<{ __typename?: 'Recipes', index: string, recipe_title: string, url: string, record_health: string, vote_count: string, rating: string, description: string, cuisine: string, course: string, diet: string, prep_time: string, cook_time: string, ingredients: string, instructions: string, author: string, tags: string, category: string, image: string, difficulty: string }> };
```

Comment:
repetition of the same recipe type, use the Recipe type you created in Types file.

do something like this: Array<Recipe & {_typename?: 'Recipes' }>.
you can define this type at the same file.

Reply:
also the file name extension should be .ts and not .tsx, we use tsx only for react components